### PR TITLE
Add missing config files

### PR DESCRIPTION
**Pull request by Tratcher**
Followup to https://github.com/aspnet/MusicStore/pull/850.

IIS Express tests are failing for OIDC and PublishAndRun. These tests didn't used to have IIS Express or nginx variations so they did not include the needed config files.

### CODE REVIEW

**Comment by Tratcher** [comment]
cc @ryanbrandenburg RE: http://aspnetci/viewLog.html?buildId=469875&tab=buildResultsDiv&buildTypeId=Lite_MusicStoreTest

File: test/MusicStore.E2ETests/nginx.conf [hunk 1]
```diff
@@ -0,0 +1,37 @@
+﻿error_log [errorlog];
```
Comment (pakrym) [inline test/MusicStore.E2ETests/nginx.conf]: Is it copied from somewhere? Can we have a default version in Hosting?

Comment (Tratcher) [reply test/MusicStore.E2ETests/nginx.conf]: Copied from server tests. Filed https://github.com/aspnet/Hosting/issues/1423

**Comment by Tratcher** [comment]
Updated. Getting NGinx to work is more involved. I've removed it for now to unblock fixing the IIS tests.

**Pull request merged**